download-aspera/README.md

```diff
@@ -0,0 +1,54 @@
+# Package download-aspera
+
+Wrapper for Aspera to download files
+
+
+## Inputs
+
+See contents of `param-file`
+
+
+## Outputs
+
+Supplied file to be downloaded
+
+
+## Usage
+
+#### Contents of Param-file
+```
+{
+    "input_file" : "/aspera-test-dir-large/100MB", ### File to be download 
```

Comment:
change `input_file` to `target_file` for consistence.

Reply:
Fixed.
https://github.com/icgc-argo/argo-data-submission/blob/ee937642d9394e5ddd35c548397905eb6ed1038b/download-aspera/tests/test-job-aspera.json#L2